linux_os/guide/system/auditing/auditd_configure_rules/audit_rules_session_events_utmp/bash/shared.sh

```diff
@@ -1,5 +1,11 @@
 # platform = multi_platform_sle,multi_platform_ubuntu
 
+{{%- if product in ['ubuntu2004'] %}}
+  {{%- set utmp_path = "/var/run/utmp" %}}
```

Comment:
`/var/run/utmp` is a symlink to `/run/utmp`
therefore this is not needed and this entire commit can be dropped